docs/get-started/authenticate-with-strava.md

```diff
@@ -0,0 +1,135 @@
+information required to set up authentication with Strava.
+
+## Step 1: Create an application in your Strava account
+
+First, create a new application in your Strava account. To do this:
+
+1. Login to your Strava account
+2. Go to settings --> My API Application
+3. Create a new application in your account
+
+:::{figure} ../images/strava-api-create-application.png
+---
+alt: "Screenshot of the Strava API create application page"
+name: strava-api-create-application
+---
+The above Strava documentation page image shows the Strava API create application page. The website value in this image is "localhost." This value can be any value if you authenticate to gain local access to your Strava data. But if you plan to build a web application, you should place the URL of your application in that field.
+:::
+
+
+:::{admonition} More resources on setting up a Strava app
+:class: tip
+
+* If you want a more technical overview, see the
+[official Strava documentation](https://developers.strava.com/docs/getting-started/#account)
+* [A helpful tutorial about
+setting up a Strava app](https://medium.com/analytics-vidhya/accessing-user-data-via-the-strava-api-using-stravalib-d5bee7fdde17)
+:::
+
+## Requesting Authorization
+
+Once you have set up your Strava API app, you are ready to authenticate.
+
+The {py:class}`stravalib.client.Client` class contains the
+{py:func}`stravalib.client.Client.authorization_url` method
+that builds an authorization URL. This URL can be clicked on by a user or used locally to
+grant your application access to Strava account data.
+
+
+:::{figure} ../images/strava_api_values.png
+---
+alt: "Screenshot of the Strava API create application page"
+name: strava-api-create-application
+---
+An image from the Strava documentation page shows what your API page will look like after you have created your app above. For the step below, you will need the Client ID value provided in your app.
+:::
+
+```python
+from stravalib import Client
+
+client = Client()
+url = client.authorization_url(
+    client_id=REPLACE_WITH_YOUR_CLIENT_ID,
+    redirect_uri="http://myapp.example.com/authorization",
+)
+```
+
+Note that you can use localhost or 127.0.0.1 as the redirect host for local development.
+
+```python
+url = client.authorization_url(
+    client_id=REPLACE_WITH_YOUR_CLIENT_ID,
+    redirect_uri="http://127.0.0.1:5000/authorization",
+)
+print(url)
+# Output:
+# 'https://www.strava.com/oauth/authorize?client_id=YOURCLIENTIDHERE&redirect_uri=http%3A%2F%2F127.0.0.1%3A5000%2Fauthorization&approval_prompt=auto&scope=read%2Cactivity%3Aread&response_type=code'
+```
+
+Once you have the URL value, you can display it in your web application to allow athletes to authorize your
+application to read their data. If you are trying to authenticate locally,
+paste the URL into your browser to support exchanging the
+temporary code Strava provides for a temporary access token.
+
```

Comment:
Maybe there should be a sentence or small paragraph here to explain that the url below is the result of Strava redirecting the browser after the user grants permissions?

Reply:
fixed!